be/src/Project/Services/Client/Client.Api/Infrastructure/ClientContext.cs

```diff
@@ -26,5 +26,6 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         public DbSet<ClientRole> ClientRoles { get; set; }
         public DbSet<ClientAccountManager> ClientAccountManagers { get; set; }
         public DbSet<ClientsAccountManagers> ClientsAccountManagers { get; set; }
+        public DbSet<ClientMarketingApproval> ClientMarketingApprovals { get; set;}
```

Comment:
**suggestion:**  @dawiddworak88 @PiotrZaranek  What you think about history of Marketing Approvals?